modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -876,16 +876,15 @@ If enabled all those relationships will be found but at the cost of lower perfor
 |false label:changed[Changed in 5.8]
 
 |--threads=<num>
-| (advanced) Max number of worker threads used by the importer. Defaults to the number of available processors reported by the JVM. There is a certain amount of minimum threads needed so for that reason there is no lower bound for this value. For optimal
-performance, this value should not be greater than the number of available processors.
+| (advanced) Max number of worker threads used by the importer. Defaults to the number of available processors reported by the JVM. There is a certain amount of minimum threads needed so for that reason there is no lower bound for this value. For optimal performance, this value should not be greater than the number of available processors.
 |20
 
 |--trim-strings[=true\|false]footnote:ingnoredByParquet2[]
 |Whether or not strings should be trimmed for whitespaces.
 |false
 
 |--update-all-matching-relationships
-|label:new[Introduced in 2025.01] If one relationship data entry matches multiple existing relationships, this decides whether to update all matching or to log as an error.
+|label:new[Introduced in 2025.01] Whether or not to update all existing relationships that match a relationship data entry. If disabled, the relationship data entry will be logged if it is within the limit of entities specified by `--bad-tolerance` and the `--skip-bad-entries-logging` option is disabled.
```

Comment:
The second sentence is not very clear because of two conditions. 'If disabled' starts and finishes the sentence. What is disabled at the beginning?

Reply:
Does it mean that when you use this option all matching relationship data entries will be updated? And if you don't use this option, then the situation is governed by `--bad-tolerance` or `--skip-bad-entries-logging` options?

Reply:
Yes, that's right.